### PR TITLE
[CDAP-12762][CDAP-12723] Bugfix Pipeline and Rule Book

### PR DESCRIPTION
**Pull request by elfenheart**
JIRAs:
- https://issues.cask.co/browse/CDAP-12762
- https://issues.cask.co/browse/CDAP-12723

_Note_:
The changes to `RuleBookDetails/index.js` is essentially moving the `rule-book-metadata` and `RulesList` into a `div`

### CODE REVIEW

File: cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss [hunk 1]
```diff
@@ -98,6 +97,13 @@ $rule-book-create-fields-required-color: $cdap-red-text;
       margin-top: -10px;
     }
   }
+
+  .rule-book-content {
+    height: calc(100% - 100px);
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss]: nit: I guess this `100px` is the height of `.rule-book-name-header`? Let's make it a variable so we only have to modify it once if we change it later

**Review by tonybach — COMMENTED** [review]
One minor comment. Besides that, code LGTM 👍

File: cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss [hunk 2]
```diff
@@ -99,7 +100,7 @@ $rule-book-create-fields-required-color: $cdap-red-text;
   }
 
   .rule-book-content {
-    height: calc(100% - 100px);
+    height: calc(100% - #{$rule-book-name-header-height});
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss]: I think you meant `$` instead of `#`

Comment (elfenheart) [reply cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss]: no, you can't just put a variable in a `calc` function

**Comment by tonybach** [comment]
Code LGTM 👍

**Review by bdmogal — APPROVED** [review]

**Pull request merged**